.github/actions/test_compat/action.yml

```diff
@@ -15,6 +15,10 @@ runs:
     - name: Maximize build space
       uses: ./.github/actions/cleanup
 
+    - name: Rust setup
```

Comment:
`dev_setup` is only needed on macos.
on linux ci,  `setup_build_tool` already made it working.

Reply:
> > @everpcpc fixed it, may help here
> 
> should be fixed with: https://github.com/datafuselabs/databend/pull/5520/files#r884142502

Is it appropriate to fetch the full history? 
The fetching time would be longer then.

Reply:
It complains in `test-compat.sh` that `wget` is not found.
What did I do wrongly?

I was trying to run it with build-tool:
`build-tool bash ./tests/compat/test-compat.sh`

`wget` should be already provided by github-action.

Reply:
ah, it's not in the build-tool image now. 
maybe you can use curl instead.

Reply:
That's all right! Thanks!